src/class/api.ga4gh.php

```diff
@@ -1286,9 +1334,7 @@ function ($aPathogenicity) {
                           vog.id, "||",
                           vog.allele, "||",
                           vog.chromosome, "||",
-                          vog.`VariantOnGenome/DNA`, "||"' .
-                    (!$bDNA38? '' : ',
-                       IFNULL(vog.`VariantOnGenome/DNA/hg38`, "")') . ', "||",
+                          vog.`VariantOnGenome/DNA' . (!$this->aActiveGBs[$sBuild]? '' : '/' . $this->aActiveGBs[$sBuild]) . '`, "||",
```

Comment:
Functionality seems to be lost here?
Did you test if the output is the same (or similar) than the output of LOVD 3.0 for a certain variant?

Reply:
Oopsie yes, if you refer to forgetting the `IFNULL()`,  it has now been added, tested and looks good!

Reply:
I assume that's what I meant indeed. Thanks!